packages/CollabCodeEdit-Server.package/CCEServer.class/instance/onError.from..st

```diff
@@ -1,5 +1,5 @@
-event handling
-onError: anException from: aSocket
-	Transcript cr; show: aSocket.
+request handling
+onError: anException from: aCCERemoteClient
+	Transcript cr; show: aCCERemoteClient.
```

Comment:
We dont just throw the error?

Reply:
Definitely not. If one client messed something up, we don't want the whole server to come crashing down